src/applications/coronavirus-vaccination/components/Introduction.jsx

```diff
@@ -22,24 +22,24 @@ function Introduction({ isLoggedIn, toggleLoginModal }) {
           step.
         </p>
       </div>
-      <p>
+      {/* <p>
         Sign up below to help us understand your interest in getting a vaccine.
         We’ll send you updates on how we’re providing vaccines across the
         country—and when you can get your vaccine if you want one. We’ll also
         offer information and answers to your questions along the way.
-      </p>
```

Comment:
Nit - I think this paragraph was okay since it says to “sign up [for updates]” instead of signing in.

Reply:
Taking a moment to read it without feeling like everything is on fire, I agree!